src/app/payment-hub/transactions/incoming-transaction-export/incoming-transaction-export.component.spec.ts

```
@@ -0,0 +1,25 @@
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ IncomingTransactionExportComponent ]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(IncomingTransactionExportComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)